### ISSUE TITLE
Cannot log into a guild server

### ISSUE THREAD

**Issue by TheWr13r** [aiko-chan-ai/DiscordBotClient#83]
hey so i tried to join my bot into a new server with friend but it couldn't due to it being a guild server, does the developer need time to update the client to the new discord layout? i'd just like to know when it's available.

**Comment by VillainsRule** [comment]
This is because bots can only join servers with OAuth, meaning the bot owner needs to generate an invite link and then give it to a server member with Manage Server or Administrator permissions. You can generate a bot invite link from Discord's [Developer Portal](https://discord.com/developers). If you want to make an automated account that can join servers, see [discord selfbots](https://www.npmjs.com/package/discord.js-selfbot-v13).

**TL;DR: Bots cannot join servers from invite links. Use the Developer Portal.**
